app/src/test/java/com/vultisig/wallet/ui/models/mappers/TokenValueToDecimalUiStringMapperTest.kt

```diff
@@ -0,0 +1,62 @@
+package com.vultisig.wallet.ui.models.mappers
+
+import com.vultisig.wallet.data.models.TokenValue
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.core.Is.`is`
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import java.math.BigDecimal
+import java.math.BigInteger
+
+public class TokenValueToDecimalUiStringMapperTest {
+
+    private lateinit var mapTokenValueToDecimalUiString: TokenValueToDecimalUiStringMapper
+
+    @BeforeEach
+    fun setUp() {
+        mapTokenValueToDecimalUiString = TokenValueToDecimalUiStringMapperImpl()
+    }
+
+    @Test
+    fun `test decimal greater than or equal to ONE_BILLION`()  {
+        val tokenValue = TokenValue(BigInteger.valueOf(3_881_138_658_686), "TOKEN", 2)
+        val result = mapTokenValueToDecimalUiString.invoke(tokenValue)
+        assertThat(
+            result,
+            `is`("38.81138659B")
+        )
```

Comment:
Why?
Because the Inspect code suggests it.
![Screenshot 2024-09-09](https://github.com/user-attachments/assets/352255ce-261e-44a3-87f4-3ad54d6aa1db)